tidy3d/components/data/monitor_data.py

```diff
@@ -209,32 +240,52 @@ def _in_plane(self) -> Dict[str, DataArray]:
             fields[field_name] = field.squeeze(dim=normal_dim, drop=True)
         return fields
 
+    @property
+    def colocation_boundaries(self) -> Coords:
+        """Coordinates to be used for colocation of the data to grid boundaries."""
+
+        if not self.grid_expanded:
```

Comment:
This was more like so that a meaningful error will be printed if it was not provided. But yeah the alternative is to make it required and remove this.